GitUI/UserControls/RevisionGrid/Columns/MessageColumnProvider.cs

```diff
@@ -219,7 +217,7 @@ public override bool TryGetToolTip(DataGridViewCellMouseEventArgs e, GitRevision
                     arrowType,
                     messageBounds,
                     e.Graphics,
-                    dashedLine: true);
+                    dashedLine: false);
```

Comment:
Please take also screenshots when a submodule is loaded. The dashed frame around superproject branches should be kept.

Reply:
Done

Reply:
Have forgot to push?

Reply:
A little more time is needed.